packages/core/src/decorators/command/command-options.ts

```diff
@@ -12,7 +12,7 @@ export interface CommandOptions {
   name: string;
 
   /**
-   * Command description
+   * Command description (Must be a blank string for context menu commands)
```

Comment:
Why blank string for context menu commands?

Reply:
That's what discord says: https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-structure